app/controllers/concerns/idv/step_indicator_concern.rb

```diff
@@ -31,14 +29,18 @@ def in_person_proofing?
     def gpo_address_verification?
       # Proofing component values are (currently) never reset between proofing attempts, hence why
       # this refers to the session address verification mechanism and not the proofing component.
-      !!current_user.pending_profile || idv_session.address_verification_mechanism == 'gpo'
+      return true if current_user&.pending_profile
```

Comment:
This should probably be 
```suggestion
      return true if current_user&.gpo_verification_pending_profile?
```